### PR TITLE
doc(atomic): add @disableStorybook tag which allows to disable props in Storybook, disable "conditions"

### PR DESCRIPTION
**Pull request by ThibodeauJF**
https://coveord.atlassian.net/browse/KIT-1509

### CODE REVIEW

**Review by olamothe — APPROVED** [review]

**Review by npushkarskii — APPROVED** [review]
LGTM! 👍

File: packages/atomic/src/components.d.ts [hunk 1]
```diff
@@ -188,7 +188,8 @@ export namespace Components {
     }
     interface AtomicFieldCondition {
         /**
-          * A list of conditions that must be fulfilled for this template to be selected.
+          * A function that must return true on results for the result template to apply.  For example, a template with the following condition only applies to results whose `title` contains `singapore`: `[(result) => /singapore/i.test(result.title)]`
+          * @disableStorybook
```
Comment (btaillon) [inline packages/atomic/src/components.d.ts:192]: Is this doc tag visible from the jsdoc/tooltips when using React, Angular or Vue?

If so, I would prefer to either add a script to remove this doc tag from those exports, or find an alternative way of declaring that a prop should not be included in storybook.

**Review by olamothe — APPROVED** [review]

File: packages/atomic/src/components.d.ts (hunk 1 above)
Comment (ThibodeauJF) [reply packages/atomic/src/components.d.ts:192]: Ah crap it's true... maybe looking at the type is a better solution

**Review by btaillon — APPROVED** [review]
I like this solution

**Pull request merged**